core/src/executor/evaluate/mod.rs

```diff
@@ -22,20 +22,40 @@ use {
     std::{borrow::Cow, rc::Rc},
 };
 
-pub use {error::EvaluateError, evaluated::Evaluated, stateless::evaluate_stateless};
+pub use {error::EvaluateError, evaluated::Evaluated};
 
 #[async_recursion(?Send)]
 pub async fn evaluate<'a, 'b: 'a, 'c: 'a, T: GStore>(
     storage: &'a T,
     context: Option<Rc<RowContext<'b>>>,
     aggregated: Option<Rc<HashMap<&'c Aggregate, Value>>>,
     expr: &'a Expr,
+) -> Result<Evaluated<'a>> {
+    evaluate_inner(Some(storage), context, aggregated, expr).await
+}
+
+pub async fn evaluate_stateless<'a, 'b: 'a>(
+    context: Option<RowContext<'b>>,
+    expr: &'a Expr,
+) -> Result<Evaluated<'a>> {
+    let context = context.map(Rc::new);
+    let storage: Option<&MockStorage> = None;
+
+    evaluate_inner(storage, context, None, expr).await
```

Comment:
```suggestion
    let context = context.map(Rc::new);
    evaluate_inner::<MockStorage>(None, context, None, expr).await
```

Is there a way to accomplish this without using MockStorage? I will consider and explore alternative options.